trunk/SUAVE/Input_Output/OpenVSP/vsp_wing.py

```diff
@@ -14,8 +14,8 @@
 
```

Comment:
This file is super annoying to review because nothing changed.

Reply:
Unintentional; I think this happened when I merged in recent changes. I had also corrected the same error in the wing tag.